react-props-and-expressions/my-app/src/App.js

```diff
@@ -0,0 +1,17 @@
+// import logo from './logo.svg';
+import './App.css';
+import CustomButton from './CustomButton';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <CustomButton text={'I'} colorClass={'red'} />
+        <CustomButton text={'know'} colorClass={'green'} />
+        <CustomButton text={'React!'} colorClass={'blue'} />
```

Comment:
You don't actually need the `{}` here! Since it's just a string. You only need to include the brackets if you're passing in a variable, or a template literal.

`<CustomButton text="I" color"red" />`